components/sankey/elements/tooltip.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Tooltip
+page_title: Sankey Tooltip
+description: Tooltip of the Sankey Diagram for Blazor.
+slug: sankey-tooltip
+tags: telerik,blazor,sankey,diagram,chart,tooltip
+published: True
+position: 9
+---
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+# Sankey Tooltip
+
+The Sankey Diagram for Blazor displays Tooltips when the user hovers the links and nodes. You can customize the rendering of these Tooltips through dedicated templates:
+* [`LinkTemplate`](#link-template)
+* [`NodeTemplate`](#node-template)
+
+To use the templates, declare a `<SankeyTooltip>` tag as a direct child of `<TelerikSankey>`. Add the desired template inside the `<SankeyTooltip>` tag. 
+
+>tip The Sankey chart also exposes the [`SquareSymbol`](/blazor-ui/api/Telerik.Blazor.Components.Sankey.Tooltip.SquareSymbol) from the default Tooltip as a standalone component in case you want to include that in the template.
+
+## Link Template
+
+The `LinkTemplate` controls the content of the Tooltip that will appear when the user hovers a link. The `NodeTemplate` exposes a `context` of type 
```

Comment:
Perhaps the content will be more easier to understand if the documentation uses plural for the tooltip word more often. The parent configuration tag is one, but the tooltips are two and are not directly related.